test/hummingbot/strategy/liquidity_mining/test_liquidity_mining.py

```diff
@@ -0,0 +1,325 @@
+        self.market_infos = market_infos
+
+        self.clock.add_iterator(self.market)
+        self.order_fill_logger: EventLogger = EventLogger()
+        self.cancel_order_logger: EventLogger = EventLogger()
+        self.market.add_listener(MarketEvent.OrderFilled, self.order_fill_logger)
+        self.market.add_listener(MarketEvent.OrderCancelled, self.cancel_order_logger)
+
+        self.default_strategy = LiquidityMiningStrategy(
+            exchange=self.market,
+            market_infos=self.market_infos,
+            token="ETH",
+            order_amount=Decimal(2),
+            spread=Decimal(0.0005),
+            inventory_skew_enabled=False,
+            target_base_pct=Decimal(0.5),
+            order_refresh_time=5,
+            order_refresh_tolerance_pct=Decimal(0.1),  # tolerance of 10 % change
+            max_order_age=3,
+        )
+
+    def simulate_maker_market_trade(
+            self, is_buy: bool, quantity: Decimal, price: Decimal, trading_pair: str, market: Optional[BacktestMarket] = None,
+    ):
+        """
+        simulate making a trade, broadcasts a trade event
+        """
+        if market is None:
+            market = self.market
+        order_book: OrderBook = market.get_order_book(trading_pair)
+        trade_event = OrderBookTradeEvent(
+            trading_pair,
+            self.clock.current_timestamp,
+            TradeType.BUY if is_buy else TradeType.SELL,
+            price,
+            quantity
+        )
+        order_book.apply_trade(trade_event)
+
+    @staticmethod
+    def has_limit_order_type(limit_orders: List[LimitOrder], trading_pair: str, is_buy: bool) -> bool:
+        for limit_order in limit_orders:
+            if limit_order.trading_pair == trading_pair and limit_order.is_buy == is_buy:
+                return True
+        return False
+
+    @unittest.mock.patch('hummingbot.strategy.liquidity_mining.liquidity_mining.estimate_fee')
+    def test_simulate_maker_market_trade(self, estimate_fee_mock):
+        """
+        Test that we can set up a liquidity mining strategy, and a trade
+        """
+        estimate_fee_mock.return_value = TradeFee(percent=0, flat_fees=[('ETH', Decimal(0.00005))])
+
+        # initiate
+        self.clock.add_iterator(self.default_strategy)
+        self.clock.backtest_til(self.start_timestamp)
+
+        # assert that there are no active trades on initialization and before clock has moved forward
+        self.assertEqual(0, len(self.default_strategy.active_orders))
+
+        # advance by one tick, the strategy will initiate two orders per pair
+        self.clock.backtest_til(self.start_timestamp + self.clock_tick_size)
+        self.assertEqual(4, len(self.default_strategy.active_orders))
+
+        # Simulate buy order fill
+        self.clock.backtest_til(self.start_timestamp + 8)
+        self.simulate_maker_market_trade(False, 50, 1, "ETH-USDT")
+        self.assertEqual(3, len(self.default_strategy.active_orders))
+
```

Comment:
Can you please a test to check if orders are created as intended? Check the order price and order size (amount) and the number of orders.